architecture/faust/dsp/libfaust-box.h

```diff
@@ -83,6 +89,42 @@ extern "C" LIBFAUST_API void createLibContext();
  */
 extern "C" LIBFAUST_API void destroyLibContext();
 
+/**
+ * Check if a box is nil.
+ *
+ * @param b - the box
+ *
+ * @return true if the box is nil, otherwise false.
+ */
+LIBFAUST_API bool isNil(Box b);
+
+/**
+ * Convert a box (such as the label of a UI) to a string.
+ *
+ * @param b - the box to convert
+ *
+ * @return a string representation of a box.
+ */
+LIBFAUST_API const char* tree2str(Box b);
```

Comment:
Hmm gotcha. This `tree2str` method is already exported with the signal API, but this PR adds the export to the Box API and adds `tree2int` as well, which you pointed out above. Would you suggest passing an error message to be filled, as the `create...` methods do?

Reply:
Also, I can just remove these `tree2...` exports from `libfaust-box`, since I can (and do) directly pull them in from `tree.hh`. However, they're still exported with `LIBFAUST_API`, but maybe it's more acceptable if they're not exposed at this level?